main.go

```diff
@@ -37,6 +39,8 @@ func main() {
 func mainError() error {
 	var err error
 
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
I think `func init()` is a better place for such stuff. I would search for it there to make sure if rand was inited.

Reply:
did not found any other `rand.Seed` so I moved it to `init` func